### ISSUE TITLE
Do the files of this crypter get detected by the Windows defender?

### ISSUE THREAD

**Issue by bill0alt** [sithis993/Crypter#110]
Hi I am having a hard time running this program because of the python compability. I just wanted to know whether it is worth it that I continue with this crypter. Does anybody know if the files of this crypter get detected by the Windows defender?

**Comment by hokocodes** [comment]
Yes, once it is opened. If you exclude the folder in Windows Security it will not be detected.
